etc/kube/pachyderm.json

```diff
@@ -325,7 +323,7 @@
                 "value": "/pach"
               },
               {
-                "name": "PFS_NUM_SHARDS",
```

Comment:
This is the actual change, I'd changed the flag pachd looks for in pachd but not in the manifest. This fixes the bug.